### PR TITLE
RHOL-357: Add `toByteArray` method to rholang.

### PR DESCRIPTION
**Pull request by goral09**
## Overview
Adds a `toByteArray` method to rholang language, similar to `nth` method. It works on arbitrary process evaluating topmost expressions before serializing. Usage:
```
@"stdout"!([10,20,30].toByteArray())
```

### Does this PR relate to an RChain JIRA issue? 
[JIRA ticket.](https://rchain.atlassian.net/browse/RHOL-357)

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

**Comment by goral09** [comment]
@guardbotmk3 We provide the `toByteArray()` method but no reverse method? I can turn any rholang code into byte array now but, besides being able to hash/encrypt/verify it, I cannot turn it back into the valid rholang program.

**Comment by guardbotmk3** [comment]
No, we're not going to create the reverse. That would break the security model of unforgeable names.

File: models/src/main/protobuf/RhoTypes.proto [hunk 1]
```diff
@@ -155,6 +155,8 @@ message Expr {
         ESet e_set_body = 22;
         EMap e_map_body = 23;
         EMethod e_method_body = 24;
+
+        bytes ByteArray = 25;
```
[the remark this answers is not in the document]
Comment (goral09) [reply models/src/main/protobuf/RhoTypes.proto]: Yeah, I've noticed that while working on adding crypto channels.

**Comment by goral09** [comment]
@guardbotmk3 updated

**Pull request merged**